src/utils/utils.ts

```diff
@@ -111,13 +111,36 @@ export const getAllScriptURLParameters = (
   return params
 }
 
-export const getTabsData = (productsData: Product[]): TabItemProps[] => {
-  const tabs = productsData.map(product => ({
-    id: product.id,
-    label: product.type,
-    contents: getProductContent(product),
-  }))
-  return tabs
+export const getProductData = (productsData: Product[]): ProductItemProps[] => {
+  const products = productsData.reduce((acc, productItem) => {
+    const indexOfProductType = acc.findIndex(
+      item => item.productType === productItem.type
+    )
+    if (indexOfProductType >= 0) {
+      acc[indexOfProductType].productItems.push({
+        id: productItem.id,
+        label: createTabLabel(productItem.termPeriod),
+        contents: getProductContent(productItem),
+      })
+    } else {
+      acc.push({
+        productType: productItem.type,
+        productItems: [
+          {
+            id: productItem.id,
+            label: createTabLabel(productItem.termPeriod),
+            contents: getProductContent(productItem),
+          },
+        ],
+      })
+    }
```

Comment:
Grouping happens here.
If `acc` already has an item with productType = productItem type, push it into the existing; else create a new grouping where productType = product item.